### PR TITLE
Update noiser for SKA

### PR DESCRIPTION
**Pull request by jiayu-google**
1. Update requirements.txt. The numpy version in the original file was outdated and caused bug.
2. Update noiser in StandardizedHistogramEstimator. Make it compatible with Gaussian and Discrete Gaussian noises.

### CODE REVIEW

**Review by kungfucraig — APPROVED** [review]

File: src/estimators/same_key_aggregator.py [hunk 1]
```diff
@@ -123,20 +127,30 @@ def __init__(self,
       epsilon_split: The proportion of total privacy budget that is assigned to
         the estimate of 1+ reach. The remaining privacy budget is assigned to
         the frequency histogram among effective keys.
+      reach_noiser_para: a dictionary of arguments other than epsilon in the
+        reach noiser. Example arguments are delta and random_state.
```
Comment (matthewclegg) [inline src/estimators/same_key_aggregator.py:131]: Could you document this a little more carefully?  E.g., specify what the data types of delta and random_state are, how they are used, and what happens if they are omitted?

File: src/estimators/same_key_aggregator.py [hunk 2]
```diff
@@ -123,20 +127,30 @@ def __init__(self,
...
+      frequency_noiser_para:  a dictionary of arguments other than epsilon in
+        the frequency noiser.
```
Comment (matthewclegg) [inline src/estimators/same_key_aggregator.py:133]: Ditto here.

File: src/estimators/tests/same_key_aggregator_test.py [hunk 3]
```diff
@@ -171,7 +174,27 @@ def test_end_to_end_estimator(self):
...
+    np.testing.assert_almost_equal(res, expected, decimal=3,
+                                   err_msg='Error for geometric noise.')
+    estimator = StandardizedHistogramEstimator(
+        max_freq=6, noiser_class=GaussianEstimateNoiser,
+        epsilon=1, epsilon_split=0.5,
+        reach_noiser_para={'delta': 1e-5,
+                           'random_state': np.random.RandomState(1)},
+        frequency_noiser_para={'delta': 1e-5,
+                               'random_state': np.random.RandomState(2)})
+    res = estimator([first_ska, second_ska, third_ska])
```
Comment (matthewclegg) [inline src/estimators/tests/same_key_aggregator_test.py:195]: I don't see any assertions here.  Am I missing something?

File: src/estimators/same_key_aggregator.py (hunk 1 above)
Comment (jiayu-google) [reply src/estimators/same_key_aggregator.py:131]: Thank you Matthew. 

Motivated by your comments + your design in the planning_eval_fw, I am working on an another way to refactor the codes. Will hopefully cook it out tomorrow, push it to another branch, and let you review.

**Comment by jiayu-google** [comment]
Close this PR as we refactor SKA in another way.

**Pull request closed without merging**